services/orchest-ctl/app/cli/start.py

```diff
@@ -70,6 +71,34 @@ def reg(
     app.start(container_config)
 
 
+@typer_app.command()
+def cloud(
+    port: Optional[int] = typer.Option(
+        8000, help="The port the Orchest webserver will listen on."
+    )
+):
+    """
+    Start Orchest in CLOUD mode.
+
+    Starting Orchest in cloud mode makes it so that the user uuid, auth
+    mode and telemetry_disabled are not modifiable through the GUI.
```

Comment:
I suggest making it into:
```suggestion
    Running Orchest in CLOUD mode changes GUI functionality. For 
    example making it impossible to disable the authentication layer.
```
because otherwise the comment might quickly become outdated.

After additional functionality it should probably become a section in the docs (as you noted).